### PR TITLE
update with last fastest release v0.2

### PR DESCRIPTION
**Pull request by nazarifard**

### CODE REVIEW

**Comment by alecthomas** [comment]
You should update the report as per the README

**Comment by deneonet** [comment]
Your fork is multiple commits behind. You need to sync your fork first before merging.

File: benchmarks.go [hunk 1]
```diff
@@ -119,6 +119,23 @@ type BenchmarkCase struct {
 }
 
 var benchmarkCases = []BenchmarkCase{
+	{
+		Name: "fastape",
+		URL:  "github.com/nazarifard/fastape",
+		New:  fastape.NewTape,
+
+		UnsafeStringUnmarshal: true,
```
Comment (matheusd) [inline benchmarks.go:127]: If fastape has both safe and unsafe unmarshalling versions, you should have the `fastape_reuse` use the unsafe version, and this one use the safe string decoding version.

**Comment by deneonet** [comment]
Closing this PR due to the author's inactivity.

**Pull request closed without merging**